examples/reference_environments/snowflake/jupyter.Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM jupyter/minimal-notebook
+FROM jupyter/minimal-notebook:python-3.10
```

Comment:
ohh you did it already in this PR @anthonyburdi 🙇🏼 thank you so much. Sorry that I'm just seeing this change now.